scripts/publish-docs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Use `set -e` instead of the `&&` chain. You can also use `set -x` (combined `set -ex`) to print the commands before executing them.